core/field_angle.js

```diff
@@ -209,32 +206,35 @@ Blockly.FieldAngle.prototype.showEditor_ = function() {
     }, svg);
   }
 
-  var border = this.sourceBlock_.getColourBorder();
-  border = border.colourBorder == null ? border.colourLight : border.colourBorder;
-
-  Blockly.DropDownDiv.setColour(this.sourceBlock_.getColour(), border);
-  Blockly.DropDownDiv.showPositionedByField(this);
   // The angle picker is different from other fields in that it updates on
   // mousemove even if it's not in the middle of a drag.  In future we may
   // change this behaviour.  For now, using bindEvent_ instead of
   // bindEventWithChecks_ allows it to work without a mousedown/touchstart.
   this.clickWrapper_ =
-      Blockly.bindEvent_(svg, 'click', this, this.hide_.bind(this));
+      Blockly.bindEvent_(svg, 'click', this, this.hide_);
   this.moveWrapper1_ =
       Blockly.bindEvent_(circle, 'mousemove', this, this.onMouseMove);
   this.moveWrapper2_ =
       Blockly.bindEvent_(this.gauge_, 'mousemove', this, this.onMouseMove);
-  this.updateGraph_();
+
+  return svg;
 };
 
 /**
- * Hide the editor and unbind event listeners.
+ * Dispose of events belonging to the angle editor.
  * @private
  */
-Blockly.FieldAngle.prototype.hide_ = function() {
+Blockly.FieldAngle.prototype.dropdownDispose_ = function() {
+  Blockly.unbindEvent_(this.clickWrapper_);
   Blockly.unbindEvent_(this.moveWrapper1_);
   Blockly.unbindEvent_(this.moveWrapper2_);
-  Blockly.unbindEvent_(this.clickWrapper_);
+};
+
+/**
+ * Hide the editor.
+ * @private
+ */
+Blockly.FieldAngle.prototype.hide_ = function() {
   Blockly.DropDownDiv.hideIfOwner(this);
   Blockly.WidgetDiv.hide();
```

Comment:
There is no WidgetDiv in an angle picker.

Reply:
Fixed.

Reply:
Ope actually there is a WidgetDiv on the angle field, because the angle field inherits from the text input field:

![AnglePicker_Hide](https://user-images.githubusercontent.com/25440652/59305161-86e4f380-8c4e-11e9-8cc8-2bb9190247d9.gif)

Readded hide.

Reply:
Nice catch!